modules/nw-operator-cr.adoc

```diff
@@ -117,6 +117,11 @@ The fields for this object specify the kube-proxy configuration.
 If you are using the OVN-Kubernetes cluster network plugin, the kube-proxy configuration has no effect.
 |====
 
+[IMPORTANT]
+====
+For a cluster that needs to deploy objects across multiple networks, ensure that you specify the same value for the `hostPrefix` parameter for each network type that is defined in the `install-config.yaml` file. Setting a different value for each `hostPrefix` parameter  can impact the CNO whereby the CNO cannot effectively route object traffic among the networks.
```

Comment:
You may want to clarify that this note is referring to clusterNetwork, rest looks good.